Snakefile

```diff
@@ -125,85 +129,73 @@ def clock_rate_std_dev(w):
     return clock_rate_std_dev[w.subtype][w.time]
 
 
-rule download:
-    message: "Downloading sequences from fauna"
+rule download_sequences:
+    output:
+        sequences = "data/all_sequences_{segment}.fasta.zst",
+    params:
+        s3_src=S3_SRC,
+    shell:
+        """
+        aws s3 cp {params.s3_src:q}/{wildcards.segment}/sequences.fasta.zst {output.sequences}
+        """
+
+rule download_metadata:
     output:
-        sequences = "data/{subtype}_{segment}.fasta"
+        metadata = "data/all_metadata.tsv.zst",

```

Comment:
It's a little easier for me to think about having the local files under `data/` have the exact same file naming as the remote files in the S3 bucket.

Reply:
That makes sense. That would look like:

- `data/all_metadata.tsv.zst` -> `data/metadata.tsv.zst`
- `data/sequences_ha.fasta.zst` -> `data/ha/sequences.fasta.zst`

Reply:
Fixed in 0b2b13d.

Reply:
Thank you!